src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,39 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img data-cy="MovieImage" src={movie.imgUrl} alt="Film logo" />
```

Comment:
More correctly to provide different alt for each movie card. Use movie title for exmpl